expo-maps/android/src/main/java/expo/modules/maps/GoogleMapsView.kt

```diff
@@ -9,7 +9,7 @@ import kotlinx.coroutines.*
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.collectLatest
 
-class GoogleMapsView(context: Context): LinearLayout(context), OnMapReadyCallback {
+class GoogleMapsView(context: Context, private val gestures: MapsViewGestures?): LinearLayout(context), OnMapReadyCallback {
 
   private val mapView: MapView = MapView(context)
   private lateinit var googleMap: GoogleMap
```

Comment:
All the other things you commented on are done the way they are because I wanted this injection through constructor. But you may be right, there is not much of implementation freedom to classes like GoogleMapsGestures, that would make anyone want to replace it with something different. In return we gain more clarity about the way things are instantiated.